Makefile

```diff
@@ -246,4 +246,82 @@ generate-schema-docs:  ## Generates markdown files for documenting the JSON Sche
 	@echo "Generating Schema docs"
 	python -m pip install "jsonschema2md"
 	python scripts/generate_docs_schemas.py
-	
\ No newline at end of file
+
+.PHONY: update_all
+update_all: ## To update all the release related files
+	@echo "The release version is: $(RELEASE_VERSION)" ; \
+	echo "The python metadata release version: $(PY_RELEASE_VERSION)" ; \
+	$(MAKE) update_maven ; \
+	$(MAKE) update_github_action_paths ; \
```

Comment:
Let's remove this from `update_all` make recipe! We will perform this action manually with make recipe call!